File: src/Emergence.Data.Shared/Search/Models/Photo.cs
using System;

namespace Emergence.Data.Shared.Search.Models
{
    public class Photo
    {
        public int Id { get; set; }
        public string Type { get; set; }
        public int? TypeId { get; set; }
        public string Filename { get; set; }
        public string BlobPath { get; set; }
        public string ContentType { get; set; }
        public int? Width { get; set; }
        public int? Height { get; set; }
        public DateTime? DateTaken { get; set; }
        public string CreatedBy { get; set; }
        public string ModifiedBy { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime? DateModified { get; set; }
        public Location Location { get; set; }
    }
}

File: src/Emergence.Web/ClientApp/src/app/shared/models/photo.ts
import { PhotoType } from "./enums";

export class Photo {
  photoId: number;
  type: PhotoType;
  typeId: number;
  filename: string;
  blobPath: string;
  contentType: string;
  width: number;
  height: number;
  dateTaken: Date;
  createdBy: string;
  dateCreated: Date;
  blobPathRoot: string;
  urlBroken: false;
  originalUri: string;
  largeUri: string;
  mediumUri: string;
  thumbnailUri: string;
  externalUrl: string;
}
